library/src/androidTest/java/com/owncloud/android/lib/resources/assistant/AssistantIT.kt

```diff
@@ -18,30 +21,53 @@ import org.junit.Test
 class AssistantIT : AbstractIT() {
     @Before
     fun before() {
-        testOnlyOnServer(NextcloudVersion.nextcloud_28)
+        testOnlyOnServer(NextcloudVersion.nextcloud_30)
```

Comment:
Why the change?
Does this mean that it will not work with NC28?

Reply:
The Text Processing API is deprecated and replaced by the Task Processing API since Nextcloud 30.

https://github.com/nextcloud/android/issues/13476